wdl-engine/tests/tasks/import-structs/source.wdl

```diff
@@ -51,9 +51,7 @@ task calculate_bill {
         currency: "Yen",
         period: "hourly"
       },
-      assay_data: {
-        "glucose": "hello.txt"
-      }
+      assay_data: {}
```

Comment:
This was removed as we're now disallowing relative path inputs in tasks and workflows as there is no defined base directory from which these paths are relative to.